posts/bonus-pokedex.md

```diff
@@ -0,0 +1,392 @@
+# How to build a Pokedex with Platformatic
+
+Recently, [Matteo Collina](https://nodeland.dev/), one of Fastify's creators and much more, launched a new Backend As A Service service: [Platformatic](https://platformatic.dev/).
+Since it is built on top of Fastify, it claims to be a life changer, and I want to try it and write down my thoughts!
+
+So, what are we going to build? A Pokedex!  
+Of course, I don't want to write a boring article, let's make it a bit more fun!  
+Here are the requirements:
+
+- The Pokemon database already exists. It will be our "legacy" system
+- I want to build a React frontend. It will test Platformatic's "extensibility". Note that Platformatic is a "Backend" framework, but fastify can handle frontend too!
+- The application must be read-only. I don't want to expose any API with `write` privileges.
+
+Now that we have the requirements let's start!
+
+
+## The database
+
+The first step is to create our database schema. Note that this section does not concern Platformatic itself,
+but I want to track my working process.
+
+The following image is the raw Entity-Relation schema for our Pokedex:
+
+![Pokedex DB Schema](./assets/pokedex-er.png)
+
+We must write down the code into a `001.do.schema.sql` file. It will be the first migration file.
```

Comment:
I think you might want to setup platformatic first, introduce migrations and then fill in the migrations file.